services/tenant-ui/frontend/src/components/innkeeper/reservation/ReservationsHistory.vue

```diff
@@ -40,6 +40,12 @@
       >
         <template #body="{ data }">
           <StatusChip :status="data.state" />
+          <RefreshPassword
+            v-if="data.state === 'approved'"
+            :id="data.reservation_id"
+            :email="data.contact_email"
+            @success="showModal"
+          />
```

Comment:
Minor styling but can we align these?

![image](https://github.com/bcgov/traction/assets/17445138/7185240d-ea49-4cd6-a917-e2fc694efb31)